### PR TITLE
Beacon CSS Tweaks

### LINKED ISSUE #170: CSS Tweaks for beacon
### Tasks
- [x] fix button issue for tablets
- [x] decrease margins for tablets, especially in portrait mode
- [ ] unsuck mobile navbar dropdown
- [x] decrease splash text size for smaller smartphones
- [x] add some padding for mobile splash
- [x] decrease margin on opportunities page
- [x] mobile table issues

### PR DESCRIPTION
**Pull request by hamhands**
### What changed
- better mobile handling (margins, font size, button width, etc.)
- responsive tables for browse opportunities
- other things in #170 
- better formatting for mobile navbar dropdown
### Issues
- closes #170
#### iPhone 4

beacon-mobile
#### iPhone 6

beacon-iphone6
#### Tablet portrait

beacon-tablet-p
#### Tablet landscape

beacon-tablet

### CODE REVIEW

File: purchasing/templates/opportunities/browse.html [hunk 1]
```diff
@@ -16,49 +16,50 @@
...
-              {% if not current_user.is_anonymous() %}
-                {% if opportunity.is_public %}
-                <td>
-                  <i class="fa fa-eye" data-toggle="tooltip" title="Visible to the public"></i>
-                </td>
-                {% else %}
-                <td>
-                  <i class="fa fa-eye-slash" data-toggle="tooltip" title="Hidden from the public"></i>
-                </td>
+        <div class="table-responsive">
```
Comment (bsmithgall) [inline purchasing/templates/opportunities/browse.html:19]: Can you make sure the indentation is right in this file?

**Comment by bsmithgall** [comment]
Can you attach screenshots as well?

**Comment by hamhands** [comment]
@bsmithgall screenshots once the navbar is done

File: purchasing/templates/opportunities/browse.html (hunk 1 above)
Comment (hamhands) [inline purchasing/templates/opportunities/browse.html:19]: Gah I thought I had. Will fix.

File: purchasing/templates/opportunities/browse.html [hunk 2]
```diff
@@ -16,49 +16,50 @@
...
               {% endif %}
-            </tr>
-            {% endif %}
-            {% endfor %}
-          </tbody>
-        </table>
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
```
Comment (hamhands) [inline purchasing/templates/opportunities/browse.html:62]: @bsmithgall yeah the table-responsive closing div is indent four, as is the opening div. They're wrapping table, just within the `if visible_active > 0` clause.

**Comment by hamhands** [comment]
@bsmithgall @shellicious COME SEE WHAT I HAVE DONE.

**Comment by hamhands** [comment]
_goes on a walk to decompress_

**Comment by msknee** [comment]
So much better, good work @hamhands!

**Comment by bsmithgall** [comment]
Looks good, but something is very screwed up with the history. You have some commits from a bad branch. This will need to be fixed before we merge otherwise it's going to cause huge headaches down the line.

**Comment by hamhands** [comment]
@bsmithgall what if I nuke this branch, checkout a new branch from master, and copy the changed files to the new branch?

**Comment by bsmithgall** [comment]
+1, just make sure you don't lose all the changes :P

**Comment by hamhands** [comment]
Moved to #176 to avoid aforementioned history horribleness.

**Pull request closed without merging**